server/controllers/interview.js

```diff
@@ -0,0 +1,109 @@
+// connect to the Interview Model
+let Interview = require('../models/interview');
+let User = require('../models/user');
+
+module.exports.createInterview = async (req, res, next) => {
+	const { guest, date, theme, questions, difficult } = req.body;
+	const user = await User.findOne({ email: req.user.user }, 'id');
+
+	let newInterview = Interview({
+		owner: user,
+		guest: guest,
```

Comment:
These should both be ObjectIds correct? At the moment it looks like user is a Mongo document and guest is a string

Reply:
Yes, those should be ObjectIds. Right before assigning those values, I am getting the user id using the following command:
```
const user = await User.findOne({ email: req.user.user }, 'id');
```
I just get rid of the guest as it is not possible from the user's perspective to create an interview with a guest. To be able to add a guest to the interview the (.../api/guest/:id) endpoint should be reached